tests/test_fields.py

```
@@ -0,0 +1,138 @@
+
+
+@Config.add_class
+class Person:
+    username: FxDCField[str] = FxDCField(
+        verbose_name="name", typechecking=True, null=False, blank=False
+    )
+    age: FxDCField[int] = FxDCField(
+        verbose_name="age", typechecking=True, null=False, blank=False
+    )
+
+    def __init__(self, username: str, age: int = 18):
```

Ruff (0.12.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)